src/modules/messages.ts

```diff
@@ -325,16 +325,15 @@ export const decodeTypeMap: DecodeTypeMapping = {
 
 export class AudioData extends Message {
   command?: AudioCommand
-  format: AudioFormat
+  decodeType: number
```

Comment:
Resolving this on the FE using mapping instead - allows us to use Format objects to reference AudioPlayers in the worker example, and minimises closed data over worker messages